src/Samples/DIPS.Xamarin.UI.Samples/Controls/Popup/ViewModel.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.ComponentModel;
+using System.Windows.Input;
+using DIPS.Xamarin.UI.Extensions;
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Samples.Controls.Popup
+{
+    public class ViewModel : INotifyPropertyChanged
+    {
+        private bool isOpen;
+
+        public ViewModel()
+        {
+            Ascending = true;
+            SaveCommand = new Command(o => Update((PopupFilterViewModel)o));
+        }
+        private void Update(PopupFilterViewModel popupFilterViewModel)
+        {
+            Ascending = popupFilterViewModel.Ascending;
+            PropertyChanged?.Raise(nameof(Ascending));
+        }
+
+        public bool IsOpen { get => isOpen; set => this.Set(ref isOpen, value, PropertyChanged); }
```

Comment:
Stick with the same extension API for property change to make it consistent for our consumers.